spec/jobs/reports/identity_verification_report_spec.rb

```diff
@@ -61,6 +62,12 @@
 
       subject.perform(report_date)
     end
+
+    it 'does not send report in email if the email field is empty' do
+      allow(IdentityConfig.store).to receive(:team_ada_email).and_return(nil)
+
+      expect(ReportMailer).to_not receive(:tables_report)
```

Comment:
This test passes without the changes. I think we're missing a call to actually perform the job?

```suggestion
      expect(ReportMailer).to_not receive(:tables_report)

      subject.perform(report_date)
```